data-es6.js

```diff
@@ -7009,6 +7009,8 @@ exports.tests = [
         es6shim: true,
         ie11: true,
         firefox2: false,
+        firefox10: true,
```

Comment:
Hmmm... Even in MDN I see that `Map` is available only from FF13... I'll check it.

Reply:
![image](https://user-images.githubusercontent.com/2213682/67440152-93c51b00-f622-11e9-87a1-33190a136b0a.png)

Reply:
Yep, and the the test fails on FF17.

Reply:
It's FF10, result - `false`.

Reply:
I have rerun the tests now and indeed it is false.  I'll fix the PR.

Reply:
I see now what I did wrong, I mixed Map and WeakMap results.  I'll rerun everything and update the PR.

Reply:
Fixed.